### PR TITLE
feat: gh-ost integration user doc

### PR DESCRIPTION
**Pull request by RainbowDashy**
Close BYT-719

### CODE REVIEW

File: content/docs/en/_layout.md [hunk 1]
```diff
@@ -93,6 +93,8 @@ expand_section_list: ["👀 Introduction", "🐣 Get Started"]
 
 ## [Multi-Tenancy](/tenant-database-management)
 
+## [gh-ost Integration](/gh-ost-integration)
```
Comment (tianzhou) [inline content/docs/en/_layout.md:96]: Let's put this under "Change Database" section, after "Migration History"

And use title "Online Schema Migration for MySQL"

File: content/docs/en/gh-ost-integration.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
+---
+title: Use gh-ost in Bytebase
+description: This guide shows you how to use gh-ost to migrate your MySQL databases in Bytebase. 
+---
+
+gh-ost is a triggerless online schema migration tool for MySQL.
```
Comment (tianzhou) [inline content/docs/en/gh-ost-integration.md:6]: let's add a link to the gh-ost github project page

File: content/docs/en/gh-ost-integration.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+All existing online-schema-change tools operate similarly: they create a ghost table in the likeness of your original table, migrate that table while empty, slowly and incrementally copy data from your original table to the ghost table, meanwhile propagating ongoing changes (any INSERT, DELETE, UPDATE applied to your table) to the ghost table. Finally, at the right time, they replace your original table with the ghost table.
+
+Bytebase uses the power of gh-ost to help users migrate MySQL databases with little downtime.
```
Comment (tianzhou) [inline content/docs/en/gh-ost-integration.md:10]:
```suggestion
Bytebase leverages gh-ost to migrate MySQL table schemas with little downtime.
```

File: content/docs/en/gh-ost-integration.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+
+<hint-block type="warning">
+
+This is currently a preview feature. Data loss is possible.
```
Comment (tianzhou) [inline content/docs/en/gh-ost-integration.md:14]:
```suggestion
This is currently a preview feature.
```

Comment (tianzhou) [reply content/docs/en/gh-ost-integration.md:14]: Let's move this to the top

File: content/docs/en/gh-ost-integration.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+
+</hint-block>
+
+## Requirements
+
+- MySQL versions 5.7 or greater.
+- Enable row-based logging.
+- Bytebase currently only supports migrating on main.
```
Comment (tianzhou) [inline content/docs/en/gh-ost-integration.md:22]: This is not clear

File: content/docs/en/gh-ost-integration.md [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+
+## Limitations
+
+- Foreign key constraints are not supported.
```
Comment (tianzhou) [inline content/docs/en/gh-ost-integration.md:26]: Should this be put under Requirement as well?

File: content/docs/en/gh-ost-integration.md [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+![A table list where "Show Bytebase reserved tables" button is highlighted](/static/docs/gh-ost-step-4-1.webp)
+![A table list which also shows reserved tables](/static/docs/gh-ost-step-4-2.webp)
+
+## Interact with gh-ost
+
+Gh-ost listens on a UNIX socket file.
+
+The UNIX socket file name is `/tmp/gh-ost.{taskID}.{databaseID}.{databaseName}.{tableName}.sock`
+
+To find the UNIX socket file, we must acquire the database id and task id.
```
Comment (tianzhou) [inline content/docs/en/gh-ost-integration.md:91]: Use "you" instead of "we" across the article.

**Review by tianzhou — CHANGES_REQUESTED** [review]

File: content/docs/en/change-database/online-schema-migration-for-mysql.md [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+
+</hint-block>
+
+[gh-ost](https://github.com/github/gh-ost) is a triggerless online schema migration tool for MySQL.
+
+All existing online schema change tools operate similarly:
+
+1. Create a ghost table in the likeness of your original table.
+1. Migrate that table while empty, slowly and incrementally copy data from your original table to the ghost table, meanwhile propagating ongoing changes (any INSERT, DELETE, UPDATE applied to your table) to the ghost table.
+1. Finally, at the right time, they replace your original table with the ghost table.
```
Comment (dragonly) [inline content/docs/en/change-database/online-schema-migration-for-mysql.md:18]:
```suggestion
1. Finally, they replace your original table with the ghost table at the right time.
```

File: content/docs/en/change-database/online-schema-migration-for-mysql.md [hunk 9]
```diff
@@ -0,0 +1,160 @@
...
+
+Bytebase leverages gh-ost to migrate MySQL table schemas with little downtime.
+
+## Requirements and limitations
+
+- MySQL versions 5.7 or greater.
+- Enable row-based logging.
+- Bytebase currently only supports migrating on primary.
```
Comment (dragonly) [inline content/docs/en/change-database/online-schema-migration-for-mysql.md:26]:
```suggestion
- Bytebase currently only supports migrating on the primary.
```

File: content/docs/en/change-database/online-schema-migration-for-mysql.md [hunk 10]
```diff
@@ -0,0 +1,160 @@
...
+- Foreign key constraints are not supported.
+- Triggers are not supported.
+
+For an exhaustive list, please refer to this [doc](https://github.com/github/gh-ost/blob/master/doc/requirements-and-limitations.md).
+
+## How to use
+
+### Step 1 - Create an alter schema issue
+
+Click “Alter Schema” on the database page.
```
Comment (dragonly) [inline content/docs/en/change-database/online-schema-migration-for-mysql.md:36]:
```suggestion
Click "Alter Schema" on the database page.
```
Let's use the straight quote mark `"` for consistency with other docs.

File: content/docs/en/change-database/online-schema-migration-for-mysql.md [hunk 11]
```diff
@@ -0,0 +1,160 @@
...
+
+![The create issue page](/static/docs/gh-ost-step-1-3.webp)
+
+### Step 2 - Approve the sync task
+
+After creating the issue, you should see something like this.
+
+![The issue detail page where gh-ost sync task is waiting approval.](/static/docs/gh-ost-step-2-1.webp)
+
+Make sure that the gh-ost sync task check is passing. Then click “Approve” to run the sync task
```
Comment (dragonly) [inline content/docs/en/change-database/online-schema-migration-for-mysql.md:56]:
```suggestion
Make sure that the gh-ost sync task check is passing. Then click “Approve” to run the sync task.
```

File: content/docs/en/change-database/online-schema-migration-for-mysql.md [hunk 12]
```diff
@@ -0,0 +1,160 @@
...
+![A table list where "Show Bytebase reserved tables" button is highlighted](/static/docs/gh-ost-step-4-1.webp)
+![A table list which also shows reserved tables](/static/docs/gh-ost-step-4-2.webp)
+
+## Interact with gh-ost
+
+Gh-ost listens on a UNIX socket file.
+
+The UNIX socket file name is `/tmp/gh-ost.{taskID}.{databaseID}.{databaseName}.{tableName}.sock`
+
+To find the UNIX socket file, you have to acquire the database id and task id.
```
Comment (dragonly) [inline content/docs/en/change-database/online-schema-migration-for-mysql.md:92]:
```suggestion
To find the UNIX socket file, you must acquire the database and task id.
```

**Review by dragonly — COMMENTED** [review]
LG otherwise.

**Review by dragonly — DISMISSED** [review]

File: content/docs/en/change-database/online-schema-migration-for-mysql.md [hunk 13]
```diff
@@ -0,0 +1,160 @@
...
+description: This guide shows you how to use gh-ost to migrate your MySQL databases in Bytebase. 
+---
+
+<hint-block type="warning">
+
+This is currently a preview feature.
+
+</hint-block>
+
+[gh-ost](https://github.com/github/gh-ost) is a triggerless online schema migration tool for MySQL.
```
Comment (tianzhou) [inline content/docs/en/change-database/online-schema-migration-for-mysql.md:12]:
```suggestion
[gh-ost](https://github.com/github/gh-ost) is a triggerless online schema migration tool for MySQL. Bytebase leverages gh-ost to migrate MySQL table schemas with little downtime.
```

File: content/docs/en/change-database/online-schema-migration-for-mysql.md [hunk 14]
```diff
@@ -0,0 +1,160 @@
...
+
+All existing online schema change tools operate similarly:
+
+1. Create a ghost table in the likeness of your original table.
+1. Migrate that table while empty, slowly and incrementally copy data from your original table to the ghost table, meanwhile propagating ongoing changes (any INSERT, DELETE, UPDATE applied to your table) to the ghost table.
+1. Finally, they replace your original table with the ghost table at the right time.
+
+Bytebase leverages gh-ost to migrate MySQL table schemas with little downtime.
```
Comment (tianzhou) [inline content/docs/en/change-database/online-schema-migration-for-mysql.md:20]: this can be removed now as it moved to the first line

**Review by tianzhou — APPROVED** [review]
LGTM

File: content/docs/en/change-database/online-schema-migration-for-mysql.md (hunk 13 above)
Comment (RainbowDashy) [reply content/docs/en/change-database/online-schema-migration-for-mysql.md]: Done

File: content/docs/en/change-database/online-schema-migration-for-mysql.md (hunk 14 above)
Comment (RainbowDashy) [reply content/docs/en/change-database/online-schema-migration-for-mysql.md]: Done

**Review by tianzhou — APPROVED** [review]
LGTM

**Review by d-bytebase — APPROVED** [review]

**Pull request merged**